app/assets/javascripts/short-form/templates/d1-preferences-programs.html.slim

```diff
@@ -13,7 +13,7 @@
 
         preference-without-proof title="{{'D1_PREFERENCES_PROGRAMS.CERT_OF_PREFERENCE' | translate}}" preference="certOfPreference" labelledby="special_programs_label" dataevent="gtm-preference-cop" description="D1_PREFERENCES_PROGRAMS.CERT_OF_PREFERENCE_DESC"
 
-        preference-without-proof title="{{'D1_PREFERENCES_PROGRAMS.DISPLACED' | translate}}" preference="displaced" labelledby="special_programs_label" dataevent="gtm-preference-dthp" description="D1_PREFERENCES_PROGRAMS.DISPLACED_TENANTS_DESC"
+        preference-without-proof title="{{'D1_PREFERENCES_PROGRAMS.DISPLACED' | translate}}" preference="displaced" labelledby="special_programs_label" dataevent="gtm-preference-dthp" description="D1_PREFERENCES_PROGRAMS.DISPLACED_TENANTS_DESC" ng-if="!listingIs('280 Brighton')""
```

Comment:
if we find the need to suppress certain preferences for many more listings before the API is ready, we can make a more generic function like `listingHasPreference` or something.